tests/garage/envs/wrappers/test_clip_reward.py

```diff
@@ -0,0 +1,25 @@
+import unittest
+
+from garage.envs.wrappers import ClipReward
+from tests.fixtures.envs.dummy import DummyRewardBoxEnv
+
+
+class TestClipReward(unittest.TestCase):
+    def test_clip_reward(self):
+        # reward = 10 when action = 0, otherwise -10
+        env = DummyRewardBoxEnv()
+        env_wrap = ClipReward(env)
```

Comment:
i'm not sure this is appropriate in a wrapper.

if an RL agent would like to clip the environment rewards, it can do that itself before processing the samples. this clipping is part of the algorithm itself, and it's important it be put right next to the rest of the algorithm code.

it seems like you are slowly building a sample preprocessing pipeline by stacking wrapper after wrapper, but that's not really the intended use of wrappers. they are just there to allow quick and algorithm-agnostic adaptation of parts of the environment interface.

Reply:
@naeioi examine this PR for an example of a preprocessing use case

Reply:
The above wrappers are all used in Atari games. Perhaps I should make them a specific function call for any Atari env, rather than environment wrappers.